src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessibility.java

```diff
@@ -817,6 +817,26 @@ public Object[] call() throws Exception {
         }, c);
     }
 
+
+    private static Accessible getAccessibleComboboxValue(Accessible a, Component c) {
```

Comment:
A comment or javadoc should mention that this is called from native code, specifically pointing to `ComboBoxAccessibility.m`.

Reply:
Done